modules/setup/application/forms/ModulePage.php

```diff
@@ -100,6 +101,8 @@ public function getModuleWizards()
         foreach ($checked as $name => $module) {
             if ($module->providesSetupWizard()) {
                 $wizards[$name] = $module->getSetupWizard();
+            } elseif (! empty($module->getRequiredModules())) {
```

Comment:
There may also be libraries required. If there's one missing a module can't be enabled as well.

Libraries don't need to be chosen, but they have to be installed. If we're now automatically checking module dependencies in the wizard, we should do the same for libraries.